src/richtext/htmlAsRichText.ts

```diff
@@ -0,0 +1,29 @@
+import rehypeParse from "rehype-parse";
+import { unified } from "unified";
+
+import { AsRichTextConfig, AsRichTextReturnType } from "./types";
+
+import { rehypeRichText } from "./utils/rehypeRichText";
+
+/**
+ * Converts an HTML string to a rich text field.
+ *
+ * @param html - An HTML string
+ * @param config - Configuration that determines the output of
+ *   `htmlAsRichText()`
+ *
+ * @returns Rich text field equivalent of the provided HTML string.
+ *
+ * @experimental - This API is subject to change and might not follow SemVer.
```

Comment:
I think my main worry is that we could realize we need those functions to be async after all, otherwise it's supposed to be stable to me.

Reply:
I see, that makes sense. Since we plan to release this helper before our work on the Migration API client, it might be better to explicitly mark the helper as unstable.

We could export the helper as `unstable_htmlAsRichText`/`unstable_markdownAsRichText`. Then, when we are more confident in the helper and its compatibility with the Migration API, we can remove `unstable_`.

I worry users won't see the `@experimental` tag in the TSDocs and expect the function to follow SemVer.

Reply:
Happy to go that way!

Reply:
Updated to `unstable_`